core/src/main/scala/scalaz/package.scala

```diff
@@ -149,9 +149,15 @@ package object scalaz {
 
   object StateT extends StateTInstances with StateTFunctions {
     def apply[F[_], S, A](f: S => F[(S, A)])(implicit F: Monad[F]): StateT[F, S, A] = IndexedStateT[F, S, S, A](f)
+    def liftM[F[_]: Monad, S, A](fa: F[A]): StateT[F, S, A] = MonadTrans[StateT[?[_], S, ?]].liftM(fa)
 
     def hoist[F[_]: Monad, G[_]: Monad, S, A](nat: F ~> G): StateT[F, S, ?] ~> StateT[G, S, ?] =
       λ[StateT[F, S, ?] ~> StateT[G, S, ?]](st => StateT((s: S) => nat(st.run(s))))
+
+    def get[F[_]: Monad, S]: StateT[F, S, S] = MonadState[StateT[F, S, ?], S].get
```

Comment:
basically just `MonadState.*` on `StateT`. Needs to be here for bincompat (not on the parent)